VagrantFile

```diff
@@ -0,0 +1,65 @@
+#to UNIX EOL
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "generic/debian10"
+  config.vm.box_version = "3.1.10"
+  config.vm.define "lever-for-change"
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  config.vm.box_check_update = true 
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine. In the example below,
+  # accessing "localhost:8080" will access port 80 on the guest machine.
+
+  # Create a private network, which allows host-only access to the machine
+  # using a specific IP.
+  config.vm.network "private_network", ip: "192.168.33.11"
+  config.vm.network "forwarded_port", guest: 80, host: 80
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
```

Comment:
Ah, if we're recommending "5000" here, then my review comment over in the `torque-sites` PR about how including an "# e.g. 5000" comment in the ansible config file there would be a good idea becomes more compelling.

(Hunh.  That sentence somehow became more complex than I intended it to be, but hopefully it parses.)

Reply:
This came from that (that setting drives this one); really this could some day be something passed into vagrant as a configurable option.